test/CappedOracle.t.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+
+import { PriceSourceMock } from "./mocks/PriceSourceMock.sol";
+
+import { CappedOracle } from "../src/CappedOracle.sol";
+
+contract CappedOracleTest is Test {
+
+    PriceSourceMock priceSource;
+
+    CappedOracle oracle;
+
+    function setUp() public {
+        priceSource = new PriceSourceMock(0.8e8);
+        oracle      = new CappedOracle(address(priceSource), 1e8);
+    }
+
+    function test_constructor() public {
+        assertEq(oracle.latestAnswer(), 0.8e8);
+        assertEq(oracle.decimals(),     8);
```

Comment:
I'd check the `source` address as well.

Reply:
Plus, not sure if that's necessary, but the test could also cover the revert on `_maxPrice <= 0`